### ISSUE TITLE
Indent specification

### ISSUE THREAD

**Issue by flowerbug** [SEIAROTg/autobean-format#9]
Hello,

I'm trying to set an alias in my shell script but it doesn't like any of my attempts at specifying something.  :(

=====
$ alias abf='autobean-format --indent 2 --currency-column 60 --cost-column 60 --output-mode inplace --thousands-separator add'
$ abf SCH.bc 
usage: autobean-format [-h] [--indent INDENT]
                       [--currency-column CURRENCY_COLUMN]
                       [--cost-column COST_COLUMN]
                       [--output-mode {stdout,diff,inplace}]
                       [--thousands-separator {add,remove,keep}]
                       [--spaces-in-braces] [--sort] [--recursive]
                       filename
autobean-format: error: argument --indent: Expected a string of spaces or tabs (not number). Got '2'.

$ alias abf='autobean-format --indent   --currency-column 60 --cost-column 60 --output-mode inplace --thousands-separator add'
$ abf SCH.bc 
usage: autobean-format [-h] [--indent INDENT]
                       [--currency-column CURRENCY_COLUMN]
                       [--cost-column COST_COLUMN]
                       [--output-mode {stdout,diff,inplace}]
                       [--thousands-separator {add,remove,keep}]
                       [--spaces-in-braces] [--sort] [--recursive]
                       filename
autobean-format: error: argument --indent: expected one argument

=====

But I have tried all sorts of other combinations of characters and escapes and have not found one that works yet for what I need (two spaces).

Please include the parameter --indent-spaces which lets someone specify the number and then I would be able to get my alias to work.

--indent-spaces 2 

Thanks!  :)

**Comment by SEIAROTg** [comment]
> autobean-format: error: argument --indent: Expected a string of spaces or tabs (not number). Got '2'.

Have you tried `--indent='  '`?

**Comment by flowerbug** [comment]
Yes, I tested it and got error messages.  I just tested it again and now it works.  I think the problem was that in my test I included backslashes to escape the single quote marks in my alias and this time I did not.

so this works:

$ alias abf="autobean-format --indent='  ' --currency-column 60 --cost-column 60 --output-mode inplace --thousands-separator add"

$ abf tst.bc
$

but this does not:

$ alias abf="autobean-format --indent=\'  \' --currency-column 60 --cost-column 60 --output-mode inplace --thousands-separator add"

$ abf tst.bc
 usage: autobean-format [-h] [--indent INDENT]
                       [--currency-column CURRENCY_COLUMN]
                       [--cost-column COST_COLUMN]
                       [--output-mode {stdout,diff,inplace}]
                       [--thousands-separator {add,remove,keep}]
                       [--spaces-in-braces] [--sort] [--recursive]
                       filename
autobean-format: error: argument --indent: Expected a string of spaces or tabs (not number). Got "'".

Thanks for the reply.  :)  Closing...
